types/src/stored_value.rs

```diff
@@ -648,6 +673,9 @@ mod serde_helpers {
         AddressableEntity(&'a AddressableEntity),
         /// Variant that stores [`BidKind`].
         BidKind(&'a BidKind),
+        /// Variant that stores [`MessageSummary`].
+        MessageTopic(&'a MessageTopicSummary),
+        Message(&'a MessageChecksum),
```

Comment:
Missing comment for `Message`?

Reply:
Fixed here: https://github.com/casper-network/casper-node/commit/3190647ec0012448907208e3a8406244433c5486